docs/tutorial/part-eight/index.md

```diff
@@ -13,51 +13,197 @@ Wow! You've come a long way! You've learned how to:
 - programmatically create pages from your data
 
-All the voices in your head are super proud of you (as am I, the anonymous tutorial instructor)! They rise in great applause! Feel free to take a moment to bow and acknolwedge their admiration 🙇
+In this final section, we're going to walk through some common steps for preparing a site to go live by introducing a powerful site diagnostic tool called [Lighthouse](https://developers.google.com/web/tools/lighthouse/). Along the way, we'll introduce a few more plugins you'll often want to use in your Gatsby sites.
 
-## What's in this tutorial?
+## Audit with Lighthouse
 
-In this final tutorial, we're going to walk through some common steps for preparing a site to go live by introducing a powerful site diagnostic tool called [Lighthouse](https://developers.google.com/web/tools/lighthouse/). Along the way, we'll introduce a few more plugins you'll often want to use in your Gatsby sites.
+Quoting from the [Lighthouse website](https://developers.google.com/web/tools/lighthouse/):
 
-## Lighthouse
+> Lighthouse is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps (PWAs), and more.
 
-Quoting from the Lighthouse website:
-
-> Lighthouse is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, and more.
-
-Lighthouse is included as part of Chrome DevTools. Running its audit and fixing problems it finds is a great way to prepare your site to go live because it helps give you confidence that your site is as fast and accessible as possible.
+Lighthouse is included in Chrome DevTools. Running its audit -- and then addressing the errors it finds and implementing the improvements it suggests -- is a great way to prepare your site to go live. It helps give you confidence that your site is as fast and accessible as possible.
 
 Let's try it out!
 
-First you need to create a production build of your Gatsby site. The Gatsby development server is optimized for making development fast but the site that it generates, while it closely resembles a production version of the site, isn't as optimized.
+First you need to create a production build of your Gatsby site. The Gatsby development server is optimized for making development fast; But the site that it generates, while closely resembling a production version of the site, isn't as optimized.
 
-Stop the development server (if it's still running) and run:
+### ✋ Create a production build
 
-`gatsby build`
+1. Stop the development server (if it's still running) and run:
 
-As you learned in part 1 of the tutorial, this does a production build
-of your site and outputs the built static files into the `public` directory.
+```bash
+gatsby build
```

Comment:
Do we have an existing resource you'd suggest linking? If there isn't one right now, I'd say let's merge, and add this later. Thoughts?

Perhaps this doc (currently a stub): https://next.gatsbyjs.org/docs/life-and-times-of-a-gatsby-build/

Reply:
Hm, perhaps the doc you mentioned as well as this one (tbh I don't even know if this would make sense here, just wondering b/c I've heard in usability tests recently that sometimes the `gatsby build` produces new errors that didn't happen during `gatsby develop`): https://next.gatsbyjs.org/docs/debugging-the-build-process/